src/boot_O2/loadfragment.c

```diff
@@ -38,29 +38,30 @@ void Load_Relocate(void* allocatedVRamAddr, OverlayRelocationSection* ovl, uintp
         reloc = ovl->relocations[i];
         relocDataP = (u32*)(sections[RELOC_SECTION(reloc)] + RELOC_OFFSET(reloc));
 
-        switch (RELOC_TYPE(reloc)) {
-            case R_MIPS_32 << 24:
+        switch (RELOC_TYPE_MASK(reloc)) {
+            case R_MIPS_32 << RELOC_TYPE_SHIFT:
```

Comment:
I don't think making defines for shifts is generally worth it over actual shifting macros most of the time, although I sort of see the point here.